src/index.html

```diff
@@ -7,7 +7,58 @@
     <link rel="stylesheet" href="./index.scss">
   </head>
   <body>
-    <h1>Hello, email client!</h1>
+    <div class="wrapper">
+      <div class="email-client">
+        <div class="email-list-container">
+          <h2 class="email-header">Inbox</h2>
+          <ul class="email-list">
+            <li class="email-item">
```

Comment:
It might be worth it to remove the hardcoded `li` tags here and generate them in the JS file instead, just as an example of how to dynamically create HTML. It shouldn't give too much away about the first task, especially since they have to figure out how to manipulate that JSON file.

Reply:
Still with the placeholder text in? I support demonstrating `document.createElement`, `document.querySelector` etc for those with more jquery background than vanilla js.

Reply:
Yeah, this ^